src/components/RadioButtons/RadioButtons.js

```diff
@@ -98,64 +64,96 @@ RadioButton.propTypes = {
   onFocus: PropTypes.func,
 }
 
-/**
- * RadioButtonGroup returns a stack of the above `<RadioButton>`'s.
- *
- * It is mostly form-library-agnostic, to lessen our dependency on redux-form,
- * but there are a few aspects worth mentioning.
- *
- * If this is used as the component for a Field in the shared/fields folder,
- * (it will), it can expect to receive a `value` prop (via redux-form). We can
- * then use this value to determine which radio button should receive the
- * `checked` prop. (See similar ideas on redux-form #124: https://git.io/fj9Tb)
- *
- * If we change form libraries/approaches, this component might need tweaks.
- *
- * @param {String}     name      The name of the field, e.g. 'faveColor'
- * @param {Array}      options   Props passed to the individual radios
- * @param {String}     value     The currently selected option.value
- * @param {Boolean}    disabled  Whether this radio group is disabled
- * @param {Boolean}    required  Whether this radio group is required
- * @param {...Object}  rest      Mostly redux-form Field input/meta props
- */
 export function RadioButtonGroup({
-  name,
   options,
-  value,
+  onChange,
```

Comment:
Other inputs do not have a public onchange handler, i don't think this one should have one. It gets called below in useEffect, and is also supplied in the form.md example.

Reply:
We have an `onSelect` for button select groups, and I believe to make these controlled inputs we need an `onChange`. Why wouldn’t we want to expose?

Reply:
well... we don't have onchange or onselect for the other inputs right? 

i don't have an opinion whether inputs should or should not have public onchange handlers, but i think the inputs should be consistent and their props should be named consistently, hence why they would all go in the same folder, and why they should share the same state-handling logic for initialvalue, currentvalue, validationSuccess callback, the gamut of form-provided props, etc. that's way out of scope for this pr so i guess you could just change this to onSelect to assuage me. still seems weird though, and pretty clearly a result of ButtonSelectGroup having the word 'select' in its name.